Makefile

```diff
@@ -37,7 +37,7 @@ help: ## Display this help.
 .PHONY: manifests
 manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
 	# Any customization needed, apply to the webhook_patch.yaml file
-	$(CONTROLLER_GEN) rbac:roleName=odh-model-controller-role,headerFile="hack/manifests_boilerplate.yaml.txt" crd webhook paths="./..." output:crd:artifacts:config=config/crd/bases
+	$(CONTROLLER_GEN) rbac:roleName=odh-model-controller-role crd webhook paths="./..." output:crd:artifacts:config=config/crd/bases
```

Comment:
I know the txt file does not have anything but it does not harm even if it is specified here. Is there any specific reason you remove this?

Reply:
for some reason if I have this in the makefile it doesn't work for me. But anyway, I added it back.